### ISSUE TITLE
content in code block goes past width of block

### ISSUE THREAD

**Issue by caleb15** [forem/forem#9475]
**Describe the bug**

Extra long lines in code blocks go past the width of the block

**To Reproduce**

1. Go to https://dev.to/bbavouzet/ubuntu-20-04-mouse-scroll-wheel-speed-536o

**Expected behavior**

Code block has a horizontal scroll

**Screenshots**

![image](https://user-images.githubusercontent.com/46505081/88327075-90f37580-ccdb-11ea-9419-92032544dea7.png)

**Desktop (please complete the following information):**

- OS: Ubuntu 18
- Browser: Chrome
- Version:  84.0.4147.89 (Official Build) (64-bit)

**Additional context**

**Comment by juliannatetreault** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by juliannatetreault** [comment]
This issue looks _very_ closely related to this issue: https://github.com/forem/forem/issues/9374

**Comment by rhymes** [comment]
Hi @caleb15 - as @juliannatetreault mentioned the issue looks like a duplicate of https://github.com/forem/forem/issues/9374 - I'm thus closing this. Would you be willing to add your comment under there?

Thank you!
